### PR TITLE
Some new backport from Luxembourg

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/+dot+travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: python
+
+python:
+- 2.7
+
+install:
+- sudo apt-get install postgis
+- sudo -u postgres createdb -E UTF8 -T template0 template_postgis
+- sudo -u postgres psql -d template_postgis -c "CREATE EXTENSION postgis;"
+- sudo -u postgres createdb -E UTF8 -T template_postgis geomapfish
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/create/+dot+travis.yml]: Is the database name always `geomapfish`?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 2]
```diff
@@ -432,7 +433,10 @@ $(PACKAGE)/locale/%/LC_MESSAGES/%/$(PACKAGE)-client.po: $(PACKAGE)/locale/$(PACK
 
 $(OUTPUT_DIR)/build.js: .build/node_modules.timestamp .build/build.js
 	mkdir -p $(dir $@)
-	awk 'FNR==1{print ""}1' $(addprefix node_modules/, jquery/dist/jquery.min.js angular/angular.min.js angular-gettext/dist/angular-gettext.min.js angular-animate/angular-animate.min.js bootstrap/dist/js/bootstrap.min.js) .build/build.js > $@
+	awk 'FNR==1{print ""}1' $(addprefix node_modules/, \
+		jquery/dist/jquery.min.js angular/angular.min.js angular-gettext/dist/angular-gettext.min.js \
+		angular-animate/angular-animate.min.js bootstrap/dist/js/bootstrap.min.js proj4/dist/proj4.js \
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: `proj4` should be specified in the project's `package.json` file.

File: c2cgeoportal/scaffolds/create/+dot+travis.yml (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+dot+travis.yml]: When we just create the project it's the case :-)
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/create/vars_%2Bpackage%2B.yaml_tmpl#L5

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Effectively thanks :-)

**Pull request merged**